client/app/about.tsx

```diff
@@ -0,0 +1,227 @@
+    id: "3",
+    about:
+      "Lorem ipsum dolor sit, amet consectetur adipisicing elit.Repellat velit alias sit earum porro modi eligendi nequeaccusantium tenetur ex rerum architecto eum cumque, explicaboofficia nihil dicta nostrum amet.",
+    image:
+      "https://trello-members.s3.amazonaws.com/6371094f58725900f3e6af44/933ac1c42df2c07ec3ad259ded59b970/original.png",
+      linkedin: 'linkedin:  linkedin/sadasgdhfdjygfugh',
+  },
+  {
+    name: "Itai Gelberg",
+    id: "4",
+    about:
+      "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat velit alias sit earum porro modi eligendi neque accusantium tenetur ex rerum architecto eum cumque, explicabo officia nihil dicta nostrum amet.",
+    image:
+      "https://www.picturecorrect.com/wp-content/uploads/2013/03/better-people-photos.jpg",
+      linkedin: 'linkedin:  linkedin/dvbkgzydsKAHLDjOSDFL',
+  },
+  {
+    name: "Shlomi azn",
+    id: "5",
+    about:
+      "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat velit alias sit earum porro modi eligendi neque accusantium tenetur ex rerum architecto eum cumque, explicabo officia nihil dicta nostrum amet.",
+    image:
+      "https://us.123rf.com/450wm/deagreez/deagreez2107/deagreez210704757/171784644-portrait-d-une-jolie-fille-gaie-montrant-une-offre-d-espace-de-copie-comme-suivre-l-abonnement-isol%C3%A9.jpg?ver=6",
+      linkedin: 'linkedin:  linkedin/GFDYSUHliasfhskusasid',
+  },
+  {
+    name: "Vladi",
+    id: "6",
+    about:
+      "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat velit alias sit earum porro modi eligendi neque accusantium tenetur ex rerum architecto eum cumque, explicabo officia nihil dicta nostrum amet.",
+    image:
+      "https://thumbs.dreamstime.com/b/close-up-happy-teenage-girl-1465128.jpg",
+      linkedin: 'linkedin:  linkedin/sadasdasasdfagsdjhadvjsd',
+  },
+  {
+    name: "Kseniya",
+    id: "7",
+    about:
+      "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat velit alias sit earum porro modi eligendi neque accusantium tenetur ex rerum architecto eum cumque, explicabo officia nihil dicta nostrum amet.",
+    image:
+      "https://www.shutterstock.com/shutterstock/photos/623101280/display_1500/stock-photo-portrait-of-serious-business-woman-standing-outside-building-623101280.jpg",
+    linkedin: 'linkedin: linkedin/sadasdhayGHEOQDWJLNRLIWEEasd',
+  },
+  {
+    name: "Tomer cohen",
+    id: "8",
+    about:
+      "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat velit alias sit earum porro modi eligendi neque accusantium tenetur ex rerum architecto eum cumque, explicabo officia nihil dicta nostrum amet.",
+    image:
+      "https://media.gettyimages.com/id/571751597/photo/portrait-of-young-business-woman-holding-paper-work-looking-away-side-view.jpg?s=612x612&w=gi&k=20&c=oMhPP2YzYQVx42_gCMGPm1R_xZE5yWLlqcSl7diBYIM=",
+      linkedin: 'linkedin:  linkedin/sadasJASDGIWAHRFKUQWHLQoilasdaidasd',
+  },
+];
```

Comment:
You can move this to MockupDatabase folder to make this file a bit shorter and easier to read.

Reply:
now is in the DBmock